src/Entities/Template/EntityDetails.jsx

```diff
@@ -132,14 +131,50 @@ const entityProperties = (obj) => {
   const attributes = obj.attributes;
   return (
     <Properties>
-      {Object.keys(attributes).map((k) => (
-        <>
-          <Text bold key={k}>
-            {capitalize(k)}:
-          </Text>
-          <PropValue>{attributes[k]}</PropValue>
-        </>
-      ))}
+      {attributes &&
+        Object.keys(attributes).map((key) => {
+          const schemaField = schema[key];
+          const value = attributes[key];
+          switch (schemaField.type) {
+            case "image":
+              return (
+                <>
+                  <Text bold key={key}>
+                    {capitalize(key)}:
+                  </Text>
+                  <PropValue>
+                    <img className="logo" src={imageUrl} alt={key} />
+                  </PropValue>
+                </>
```

Comment:
The `key` property should be added at the top level. This means this section should be look like this:
```suggestion
                <Fragment key={key}>
                  <Text bold>
                    {capitalize(key)}:
                  </Text>
                  <PropValue>
                    <img className="logo" src={imageUrl} alt={key} />
                  </PropValue>
                </Fragment>
```
`<Fragment>` and `<>` are both fragments. I'm fine with moving on with this code, but it would be good to have this change as a separate PR

Reply:
Nice catch, thanks! I'll do it in a separate PR to keep main & develop in sync.